deployer/scripts/functions.sh

```diff
@@ -30,11 +30,23 @@ function setup_certificate {
 function handle_previous_deployment() {
   if [ "$mode" = "refresh" ]; then
     echo "Deleting any previous deployment (leaving route and PVCs)"
-    oc delete rc,svc,pod,sa,templates,secrets --selector="metrics-infra"
+    oc delete rc,svc,pod,sa,templates,secrets --selector="metrics-infra" --ignore-not-found=true
```

Comment:
Ah shoot, yes, it will. I didn't update this section as since it doesn't have --grace-period=0 which causes the issue. New PR incoming